components/inputs/docs/input-select-styles.md

```diff
@@ -48,60 +48,159 @@ Toggling progressive disclosure is OK
 
-Native `<select>` elements can be styled by importing `input-select-styles.js` into your LitElement and applying the `d2l-input-select` CSS class.
+Native `<select>` elements can be styled by importing `input-select-styles.js` into your LitElement and applying the `d2l-input-select` CSS class. The styles account for several CSS pseudo-classes, including `disabled`, `focus` and `hover`.
 
-Note: in order for RTL to function correctly, make sure your component uses the `RtlMixin`.
+<!-- docs: demo code name:d2l-test-input-select -->
+```html
+<script type="module">
+  import { html, LitElement } from 'lit';
+  import { selectStyles } from '@brightspace-ui/core/components/inputs/input-select-styles.js';
+
+  class TestInputSelect extends LitElement {
+  static get styles() {
+    return [ selectStyles ];
+  }
+
+  render() {
+    return html`
+      <select
+        aria-label="Choose a dinosaur:"
+        class="d2l-input-select">
+        <option>Tyrannosaurus</option>
+        <option>Velociraptor</option>
+        <option>Deinonychus</option>
+      </select>
+    `;
+  }
+
+  }
+  customElements.define('d2l-test-input-select', TestInputSelect);
+</script>
+<d2l-test-input-select></d2l-test-input-select>
+```
+
+## Building a Custom Select Component
+
+In addition to the `d2l-input-select` styling and [native attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#attributes), there are a few suggested Mixins and properties that we recommend using when creating a custom `Select` component.
+
+### Invalid
+
+Through the use of the [`aria-invalid`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-invalid) attribute, you are able to mark the input as invalid, which shows up both in the styling along with being picked up by screenreaders.
+
+<!-- docs: demo name:d2l-test-input-select -->
+```html
+<script type="module">
+  import { html, LitElement } from 'lit';
+  import { selectStyles } from '@brightspace-ui/core/components/inputs/input-select-styles.js';
+
+  class TestInputSelect extends LitElement {
+
+    static get properties() {
+    return {
+      invalid: { type: Boolean }
+    };
+  }
+
+  static get styles() {
+    return [ selectStyles ];
+  }
+
+  render() {
+    const invalid = this.invalid ? 'true' : 'false';
+    return html`
+      <select
+        aria-label="Choose a dinosaur:"
+        aria-invalid="${invalid}"
+        class="d2l-input-select">
+        <option>Tyrannosaurus</option>
+        <option>Velociraptor</option>
+        <option>Deinonychus</option>
+      </select>
+    `;
+  }
+
+  }
+  customElements.define('d2l-test-input-select', TestInputSelect);
+</script>
+<d2l-test-input-select invalid></d2l-test-input-select>
+```
 
-<!-- docs: demo code properties name:d2l-test-input-select -->
+### RtlMixin
+
+The [`RtlMixin`](https://github.com/BrightspaceUI/core/tree/main/mixins/rtl) is ideal for supporting right-to-left languages.
```

Comment:
Maybe we should be more direct and say 

> Use the [`RtlMixin`](https://github.com/BrightspaceUI/core/tree/main/mixins/rtl) to support languages that read from right to left.

Reply:
Actually, we [shouldn't be recommending new usages of RtlMixin](https://blog.d2l.dev/2024/05/31/state-of-the-ui/#right-to-forget-rtlmixin) at all, in favour of CSS logical properties.

Reply:
Got it, I'll remove the section about it then!